src/extensions/default/JavaScriptQuickEdit/unittest-files/jquery-ui/tests/unit/widget/widget_core.js

```diff
@@ -162,7 +162,7 @@ test( "direct usage", function() {
 			ok( shouldCreate, "create called on instantiation" );
 		},
 		methodWithParams: function( param1, param2 ) {
-			ok( true, "method called dirctly" );
+			ok( true, "method called directly" );
```

Comment:
jQuery-ui is a third party dependency so these changes should be posted in https://github.com/jquery/jquery-ui